pagopa/pn-frontend
# PN-8864-fe-toast-non-vengono-rimossi

## Short Description
This fix ensures that the logout action is triggered when the SessionModal appears due to unauthorized user, Terms of Service API errors, or a closed session. Additionally, it clears the sessionStorage in these cases to maintain a clean state.

## List of Changes Proposed in This Pull Request
- Added `dispatch(logout())` to the `handleClose()` method in the SessionModal to clear the sessionStorage.

## How to Test
1. Run `yarn start` in the project.
2. Simulate an error/success snackbar message.
3. Check if the message persists when you log in again.
